passive/s3.js

```diff
@@ -0,0 +1,40 @@
+// S3 bucket finder by alishasinghania09@gmail.com 
+
+function scan(ps, msg, src) {
+    // populate some parameters which will be needed if s3 bucket url is present
+    var alertRisk = 1
+    var alertReliability = 3
```

Comment:
```suggestion
    var alertConfidence= 3
```

I'm sure you had copied this from other examples, we're trying to get things cleaned up.

If you are/were having problems with squashing or amending commits just let me know I don't mind doing the change for you.

Reply:
It was suggested in the previous review too https://github.com/zaproxy/community-scripts/pull/213#discussion_r602141056